csharp/src/Drivers/BigQuery/BigQueryConnection.cs

```diff
@@ -994,7 +1012,7 @@ public override AdbcStatement CreateStatement()
         {
             if (this.credential == null)
             {
-                throw new InvalidOperationException();
+                throw new InvalidOperationException("A credential must be set");
```

Comment:
Should this be an `AdbcException`? We could set the status code to `AdbcStatusCode.Unauthenticated`.

Reply:
updated in latest PR